controls/SV-221588.rb

```diff
@@ -56,4 +56,21 @@
   tag legacy: ["SV-94635","V-79931"]
   tag cci: ["CCI-000169"]
   tag nist: ["AU-12 a"]
+
+  if input('siprnet_system') == 'true'
+    impact 0.0
+    describe 'This Control is Not Applicable to systems on the SIPRNet.' do
+      skip 'This Control is Not Applicable to systems on the SIPRNet.'
+    end
+  else
+    describe.one do
+        describe registry_key('HKEY_LOCAL_MACHINE\Software\Policies\Google\Chrome') do
+          it { should have_property 'DownloadRestrictions' }
+          its('DownloadRestrictions') { should cmp 1 }
+        end
+        describe registry_key('HKEY_LOCAL_MACHINE\Software\Policies\Google\Chrome') do
+          it { should have_property 'DownloadRestrictions' }
+          its('DownloadRestrictions') { should cmp 2 }
+        end
+     end
```

Comment:
An `end` is missing for this `else` block. Consider adding an `end` on this line to fix the syntax.